### ISSUE TITLE
MatConvNet error keep going in the same horizontal line (it doesn't decrease)

### ISSUE THREAD

**Issue by kamelgithub** [vlfeat/matconvnet#545]
I am using matconvnet for regression task to detect human pose from an image. I always get the same output pose  for different input when I test the model , the error graph during training does'nt decrease, but the objective graph decreases.

After 5 times call to the loss function during the training process , it start producce nan values.

I changed the learning rate , weight decay, momuntum to different values ,but I still have the same problem.

this is my loss function:

///////////////////////////////////////////////////////

function Y = vl_nnL2(X,c,dzdy)
if nargin <= 2 
Y = 0.5*sum(((X)-c).^2);
else
Y = +(X-c)*dzdy;
Y = reshape(Y,size(X));
end

Even I change it to softmaxloss, I get the same NaN values.

My input data is  224*224*1  image binary image
output : (1*1*75)              // 25*3d points  for each input image.

when I decrease the learning rate I get different output for different input, but each output is very far from the ground truth (not like human body shape).

when I increase the learning rate I get same output for different input, but this output is corresponding to one of the training data ( human body shape).

**Comment by jellyzhang** [comment]
I have same problem in fcn, have you solved it ?

**Comment by kamelgithub** [comment]
not yet.
